project-base/tests/FrontendApiBundle/Functional/Order/MultipleProductsInOrderTest.php

```diff
@@ -5,21 +5,23 @@
 namespace Tests\FrontendApiBundle\Functional\Order;
 
 use App\DataFixtures\Demo\ProductDataFixture;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
 
 class MultipleProductsInOrderTest extends AbstractOrderTestCase
 {
     public function testCreateFullOrder(): void
     {
+        $firstDomainLocale = $this->domain->getDomainConfigById(Domain::FIRST_DOMAIN_ID)->getLocale();
         $expected = [
             'data' => [
                 'CreateOrder' => [
                     'transport' => [
-                        'name' => 'Czech post',
+                        'name' => t('Czech post', [], 'dataFixtures', $firstDomainLocale),
                     ],
                     'payment' => [
-                        'name' => 'Cash on delivery',
+                        'name' => t('Cash on delivery', [], 'dataFixtures', $firstDomainLocale),
                     ],
-                    'status' => 'New',
+                    'status' => t('New [adjective]', [], 'dataFixtures', $firstDomainLocale),
                     'totalPrice' => [
                         'priceWithVat' => '4964.44',
                         'priceWithoutVat' => '4103.19',
```

Comment:
You will also need to updated all prices in all tests like this:
```suggestion
                        'priceWithVat' => $this->getPriceWithVatConvertedToDomainDefaultCurrency('4964.44'),
                        'priceWithoutVat' => $this->getPriceWithVatConvertedToDomainDefaultCurrency('4103.19'),
```